javascript-loops-practice/get-student-names.js

```diff
@@ -0,0 +1,13 @@
+/* exported getStudentNames */
+function getStudentNames(students) {
+  var nameArray = [];
+  var i = 0;
+  var object = students[i];
+  for (i; i < students.length; i++) {
+    for (var key in object) {
+      nameArray.push(object[key]);
+      object = students[i + 1];
+    }
```

Comment:
Why the loop? You don't need to do an action for every single property of the object, this function just wants the value of the `name` property

Reply:
Thanks for all the pointers, seems i was doing more than i needed for a lot of these. Thank you!!